### PR TITLE
Add Github Corner

### PR DESCRIPTION
**Pull request by ita07**
This is my first PR, hope this is what you are looking for!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,13 @@
   </head>
   <body>
     <input type="url" placeholder="Type a URL" autofocus autocomplete/>
+    <a href="https://github.com/tholman/github-corners" class="github-corner" aria-label="View source on GitHub">
```
Comment (pavi2410) [inline index.html]: Update the href value

**Comment by pavi2410** [comment]
Can you please move it after the table tag after line 25

**Comment by ita07** [comment]
I updated it with your requests!

**Comment by pavi2410** [comment]
Looks good to me!
http://htmlpreview.github.io/?https://github.com/ita07/URL-Inspector/blob/9e9638cadd6d0d8f4a5a1e1692d3f795f01f72b2/index.html

**Comment by pavi2410** [comment]
Thank you 😀😚

**Comment by pavi2410** [comment]
Closes #9

**Pull request merged**